### ISSUE TITLE
Daemon dependencies only compile with Go 1.10.*

### ISSUE THREAD

**Issue by ccyanxyz** [singnet/snet-daemon#201]
I got this error when I try to build snetd on Ubuntu 16.04:
./scripts/build linux amd64
github.com/singnet/snet-daemon/vendor/github.com/ethereum/go-ethereum/crypto/bn256/cloudflare.gfpMul: relocation target runtime.support_bmi2 not defined

go version go1.11.4 linux/amd64

Any help would be appreciated!

**Comment by vsbogd** [comment]
Hi @ccyanxyz, I have seen this before, as far as I remember as fast workaround using `go` version 1.10 (for example go1.10.4 in Ubuntu) should help. But it still should be clarified how to make it possible compiling `snet daemon` with go1.11 or both.

**Comment by ccyanxyz** [comment]
Thanks, I'll give it a try.

**Comment by ferrouswheel** [comment]
Updating to a newer version of go-ethereum should resolve this https://github.com/ethereum/go-ethereum/issues/17237

**Comment by vsbogd** [comment]
As far as I remember last version of go-ethereum has bug in abigen which breaks contract api generation. I have raised PR to go-ethereum repo couple of weeks ago, but it is still not merged. I have pinged guys yesterday, will see if it will help.

**Comment by vsbogd** [comment]
Link to my go-ethereum PR https://github.com/ethereum/go-ethereum/pull/19207

**Comment by vsbogd** [comment]
Raised separate issue in go-ethereum https://github.com/ethereum/go-ethereum/issues/19292

**Comment by vsbogd** [comment]
To not forget: after this issue is fixed we can remove notify patch see: https://github.com/singnet/snet-daemon/pull/230/files#r267251251

**Comment by vsbogd** [comment]
I have migrated code to Go modules and Go 1.15 in https://github.com/singnet/snet-daemon/pull/519
It is blocked by #518
